src/main/java/frc/robot/Constants.java

```diff
@@ -39,4 +39,23 @@ public static class WheelConstants {
     
 
   }
+  //All measurements in inches
+  public static class ArenaConstants {
+    public static final double midCubeHeight = 18.25;
+    public static final double highCubeHeight = 35.5;
+    public static final double midCubeFromFront = 14.25;
+    public static final double highCubeFromFront = 31.625;
+
```

Comment:
Some of these names are not very clear.
Follow naming convention of k<name><Units>. 
Make sure all units are in meters. This can be done with Units.inchesToMeters